pkg/scripts_test/check.go

```diff
@@ -250,3 +251,13 @@ func checkAbortedDueToDifferentTags(c check) {
 	require.Greater(c.test, len(c.output), 0)
 	require.Contains(c.test, c.output[len(c.output)-1], "You are trying to install with different tags than in your configuration file!")
 }
+
+func checkUserExists(c check) {
+	_, err := user.Lookup("otelcol-sumo")
```

Comment:
nit: Would not make sense extract this string as const?

Reply:
I'll change it